packages/components/tabs/src/tabs.tsx

```diff
@@ -87,13 +87,17 @@ export default defineComponent({
 
     const nav$ = ref<TabNavInstance>()
     const panes = reactive<Record<number, TabsPaneContext>>({})
-    const currentName = ref<TabPanelName>(
-      props.modelValue ?? props.activeName ?? '0'
-    )
+    const currentName = computed<TabPanelName>({
+      get() {
+        return props.modelValue ?? props.activeName ?? '0'
+      },
+      set(value) {
+        emit(UPDATE_MODEL_EVENT, value)
+      },
+    })
 
     const changeCurrentName = (value: TabPanelName) => {
       currentName.value = value
```

Comment:
I think the problem is here, the currentName value should not be modified inside the component

Reply:
if we use dynamic props instead of v-model,the tab will not switch when we click nav

Reply:
test failed

Reply:
sorry,this issue is out of my depth,I will close this pr,thank you